src/main/java/org/dependencytrack/policy/cel/compat/VersionDistanceCelScriptBuilder.java

```diff
@@ -6,7 +6,19 @@ public class VersionDistanceCelScriptBuilder implements CelPolicyScriptSourceBui
     @Override
     public String apply(PolicyCondition policyCondition) {
         return """
-                component.version_distance("%s", "%s")
-                    """.formatted(policyCondition.getOperator(), CelPolicyScriptSourceBuilder.escapeQuotes(policyCondition.getValue()));
+                component.version_distance("%s", %s)
+                    """.formatted(comparator(policyCondition.getOperator()), policyCondition.getValue());
```

Comment:
To ensure compatibility with legacy conditions, we still need to expect `policyCondition.getValue()` to return JSON. The frontend still submits JSON when configuring the condition via UI, and users configuring policies via REST API will also still send JSON. The script builder's job would then be to map the provided JSON to the Protobuf struct.